src/ProblemBrowser/ProblemBrowserResults.tsx

```diff
@@ -0,0 +1,283 @@
+                                // This problem is the original problem
+                                ...problem,
+                                // with the path switched with each additional path
+                                webworkQuestionPath: additionalProblemPath
+                            }))
+                        ]
+                    ));
+                    const problemObjects = allProblems.map((problem) => ({
+                        path: problem.webworkQuestionPath,
+                        meta: {
+                            type: ProblemBrowserSearchType.COURSE,
+                            topicName: problem.topic?.name,
+                            unitName: problem.topic?.unit?.name,
+                            courseName: problem.topic?.unit?.course?.name
+                        }
+                    }));
+                    setProblemDictionary({
+                        problems: _.keyBy(problemObjects, 'path'),
+                    });
+                    break;
+                }
+                default:
+                    logger.warn('ProblemBrowserResults: Invalid type, either a bug or someone is manipulating the url');
+                    setProblemDictionary({
+                        problems: {}
+                    });
+                    break;
+                }        
+            } catch (e) {
+                logger.error(e);
+                setUpdateAlert({message: e.message, severity: 'error'});
+                setProblemDictionary({
+                    problems: {}
+                });
+            }
+        })();
+    }, [searchType]);
+
+    if (_.isNil(problems) || _.isNil(problemDictionary)) {
+        return <div>Loading...</div>;
+    }
+
+    if (_.isEmpty(problems)) {
+        return <div>
+            <Snackbar
+                anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+                open={updateAlert.message !== ''}
+                autoHideDuration={updateAlert.severity === 'success' ? 6000 : undefined}
+                // onClose={() => setUpdateAlert(alertState => ({...alertState, message: ''}))}
+                style={{maxWidth: '50vw'}}
+            >
+                <MUIAlert
+                    // onClose={() => setUpdateAlert(alertState => ({...alertState, message: ''}))}
+                    severity={updateAlert.severity}
+                    variant='filled'
+                    style={{fontSize: '1.1em'}}
+                >
+                    {updateAlert.message}
+                </MUIAlert>
+            </Snackbar>
+            There are no problems to display
+        </div>;
+    }
+
+    if (_.isNil(selectedProblem)) {
+        return <div>Loading...</div>;
+    }
+
+    return (
+        <>
+            {/* {alert.message !== '' && <Alert severity={alert.severity}>{alert.message}</Alert>} */}
+            <Container fluid style={{
```

Comment:
Bleh, Bootstrap. But I guess we don't have a `pills` alternative yet.

Reply:
Yeah, right now my main goal was to keep it as consistent as possible to the original simple problem page